pages/implementation/start-using-avo-functions.mdx

```diff
@@ -41,45 +41,22 @@ If a source doesn't already exist for your app or website, click "Add Source" to
 
 ### Configure destinations
 
-In Avo there are two high level options when it comes to destinations. We'll cover the difference and how you can set your destination up in your Avo workspace:
-1. [Option 1: Self-managed analytics SDK instances (custom destination)](#option-1-initialize-self-managed-analytics-sdk-instance)
-2. [Option 2: Fully Avo managed analytics SDK](#option-2-fully-avo-managed-destinations)
-3. [Configure in your Avo workspace](#configure-in-your-avo-workspace)
+1. [Provide the destination interface for each analytics destination](#implement-the-destination-interface)
+2. [Configure in your Avo workspace](#configure-in-your-avo-workspace)
```

Comment:
Shouldn't this be reversed, configure in the workspace first, then implement the interface in code?

Reply:
Fixed in a85fd0b